docs/source/releases/v2.0.rst

```diff
@@ -223,7 +233,7 @@ Backwards incompatible changes in Oscar 2.0
 - Changed the values returned by the Oscar app config ``urls`` property. It now
   returns a tuple containing the list of URL patterns, the app namespace (which
   could previously be None, but not any more), and the instance namespace
-  (which would previously be overiden by the app namespace, if left blank, but
+  (which would previously be overidden by the app namespace, if left blank, but
   must now be explicitly set). To include URLs with an instance namespace, use
   the form ``app_config.urls``, and to include URLs without an instance
   namespace, use the form ``django.conf.urls.include(app_config.urls[0])``.
```

Comment:
I can't comment on the line below, because it's not part of the diff, but  :meth:\`\`oscar.get_core_apps\`\` doesn't work because that method has been removed. We need to drop the :meth: